README.md

```diff
@@ -1,2 +1,132 @@
 # sms-aws-sqs-python-receiver
-Python Flask AWS function to receive bulk SMS messages and store in AWS SQS.
+Python function to store bulk messages to AWS SQS without consideration of source.
+
+## Prerequisites
+* Python 3.8 (update `serverless.yml` if higher version is desired)
+* Pip
+* [Node.js](https://nodejs.org/en/) and npm
+* [Serverless Framework](https://serverless.com/framework/docs/getting-started/)
+* [AWS account](https://aws.amazon.com/)
+
+## Setup Instructions
+Clone this repo from GitHub, and navigate into the newly created directory to proceed.
+
+### Environment
+Rename `.env.default` to `.env` and add values to `VONAGE_API_KEY` and `VONAGE_API_SECRET` provided by your [Vonage API Dashboard](https://dashboard.nexmo.com/).
```

Comment:
Have we settled on renaming the default API key and secret variables to `VONAGE_API_KEY` and `VONAGE_API_SECRET` formally? I thought they were still to be named `NEXMO_API_KEY` and `NEXMO_API_SECRET` since there is actually no "Vonage API" yet but rather a collection of API services (Nexmo, OpenTok, etc.) each with their own non-unified API credentials.